ddtrace/internal/utils/time.py

```diff
@@ -1,20 +1,44 @@
 from datetime import datetime
+from datetime import timedelta
 from types import TracebackType
 from typing import Optional
 from typing import Type
 
 from ddtrace.internal import compat
+from ddtrace.internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+def fromisoformat_py2(t):
```

Comment:
Does it make sense to put this function in the `compat` module (and import it from there as well)?

Reply:
I am not sure it is 100% compatible with the Python 3 implementation, so being only in here for now is probably ok since this is specific for this response data.

Making compatible with Python 3 implementation and moving to compat could be a good follow-up.